### ISSUE TITLE
Network slow, powered devices unresponsive on latest-dev. Switch to latest does not help

### ISSUE THREAD

**Issue by adorobis** [Koenkk/zigbee2mqtt#16846]
### What happened?

Since this morning the network is very slow, mains powered devices are unresponsive (e.g. can't switch a smart plug), battery powered remotes don't work at all or only occasionally. 
I've been using so far the latest-dev channel after quite similar issue in the past, since then it was rock solid until today. I've switched to the `latest` tag, rebooted the physical host running the zigbee container, unplugged some of the devices and no improvement so far. Sensors (e.g. door ones) are not always updated.

### What did you expect to happen?

Just normal operation of all devices 

### Zigbee2MQTT version

1.30.1

### Adapter firmware version

0x26780700

### Adapter

Conbee II

### Debug log

Switched again to latest-dev and here is the debug log.
[zigbee_logs.txt](https://github.com/Koenkk/zigbee2mqtt/files/10848550/zigbee_logs.txt)

**Comment by adorobis** [comment]
Interesting observation, in my case it seems that a Moes BHT-002-GCLZB termostat is flooding the network with messages:
```
debug 2023-02-28 13:56:06: Received Zigbee message from 'Termostat Szary', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,22],"type":"Buffer"},"datatype":2,"dp":16}],"seq":25091}' from endpoint 1 with groupID null
info  2023-02-28 13:56:06: MQTT publish: topic 'zigbee2mqtt/Termostat Szary', payload '{"child_lock":"UNLOCK","current_heating_setpoint":22,"deadzone_temperature":1,"heat":"ON","linkquality":255,"local_temperature":21,"local_temperature_calibration":-2,"max_temperature":35,"max_temperature_limit":26,"min_temperature_limit":null,"preset":"program","preset_mode":"program","program":{"saturday_p1_hour":6,"saturday_p1_minute":0,"saturday_p1_temperature":21,"saturday_p2_hour":12,"saturday_p2_minute":0,"saturday_p2_temperature":21,"saturday_p3_hour":14,"saturday_p3_minute":0,"saturday_p3_temperature":21,"saturday_p4_hour":19,"saturday_p4_minute":0,"saturday_p4_temperature":19,"sunday_p1_hour":8,"sunday_p1_minute":0,"sunday_p1_temperature":21,"sunday_p2_hour":12,"sunday_p2_minute":0,"sunday_p2_temperature":21,"sunday_p3_hour":14,"sunday_p3_minute":0,"sunday_p3_temperature":21,"sunday_p4_hour":19,"sunday_p4_minute":0,"sunday_p4_temperature":19,"weekdays_p1_hour":4,"weekdays_p1_minute":30,"weekdays_p1_temperature":21,"weekdays_p2_hour":12,"weekdays_p2_minute":0,"weekdays_p2_temperature":21,"weekdays_p3_hour":15,"weekdays_p3_minute":0,"weekdays_p3_temperature":21,"weekdays_p4_hour":19,"weekdays_p4_minute":0,"weekdays_p4_temperature":19},"running_state":"heat","sensor":"IN","system_mode":"heat"}'
debug 2023-02-28 13:56:06: Received Zigbee message from 'Termostat Szary', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,22],"type":"Buffer"},"datatype":2,"dp":16}],"seq":25347}' from endpoint 1 with groupID null
info  2023-02-28 13:56:06: MQTT publish: topic 'zigbee2mqtt/Termostat Szary', payload '{"child_lock":"UNLOCK","current_heating_setpoint":22,"deadzone_temperature":1,"heat":"ON","linkquality":255,"local_temperature":21,"local_temperature_calibration":-2,"max_temperature":35,"max_temperature_limit":26,"min_temperature_limit":null,"preset":"program","preset_mode":"program","program":{"saturday_p1_hour":6,"saturday_p1_minute":0,"saturday_p1_temperature":21,"saturday_p2_hour":12,"saturday_p2_minute":0,"saturday_p2_temperature":21,"saturday_p3_hour":14,"saturday_p3_minute":0,"saturday_p3_temperature":21,"saturday_p4_hour":19,"saturday_p4_minute":0,"saturday_p4_temperature":19,"sunday_p1_hour":8,"sunday_p1_minute":0,"sunday_p1_temperature":21,"sunday_p2_hour":12,"sunday_p2_minute":0,"sunday_p2_temperature":21,"sunday_p3_hour":14,"sunday_p3_minute":0,"sunday_p3_temperature":21,"sunday_p4_hour":19,"sunday_p4_minute":0,"sunday_p4_temperature":19,"weekdays_p1_hour":4,"weekdays_p1_minute":30,"weekdays_p1_temperature":21,"weekdays_p2_hour":12,"weekdays_p2_minute":0,"weekdays_p2_temperature":21,"weekdays_p3_hour":15,"weekdays_p3_minute":0,"weekdays_p3_temperature":21,"weekdays_p4_hour":19,"weekdays_p4_minute":0,"weekdays_p4_temperature":19},"running_state":"heat","sensor":"IN","system_mode":"heat"}'
debug 2023-02-28 13:56:06: Received Zigbee message from 'Termostat Szary', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,22],"type":"Buffer"},"datatype":2,"dp":16}],"seq":25603}' from endpoint 1 with groupID null
info  2023-02-28 13:56:06: MQTT publish: topic 'zigbee2mqtt/Termostat Szary', payload '{"child_lock":"UNLOCK","current_heating_setpoint":22,"deadzone_temperature":1,"heat":"ON","linkquality":255,"local_temperature":21,"local_temperature_calibration":-2,"max_temperature":35,"max_temperature_limit":26,"min_temperature_limit":null,"preset":"program","preset_mode":"program","program":{"saturday_p1_hour":6,"saturday_p1_minute":0,"saturday_p1_temperature":21,"saturday_p2_hour":12,"saturday_p2_minute":0,"saturday_p2_temperature":21,"saturday_p3_hour":14,"saturday_p3_minute":0,"saturday_p3_temperature":21,"saturday_p4_hour":19,"saturday_p4_minute":0,"saturday_p4_temperature":19,"sunday_p1_hour":8,"sunday_p1_minute":0,"sunday_p1_temperature":21,"sunday_p2_hour":12,"sunday_p2_minute":0,"sunday_p2_temperature":21,"sunday_p3_hour":14,"sunday_p3_minute":0,"sunday_p3_temperature":21,"sunday_p4_hour":19,"sunday_p4_minute":0,"sunday_p4_temperature":19,"weekdays_p1_hour":4,"weekdays_p1_minute":30,"weekdays_p1_temperature":21,"weekdays_p2_hour":12,"weekdays_p2_minute":0,"weekdays_p2_temperature":21,"weekdays_p3_hour":15,"weekdays_p3_minute":0,"weekdays_p3_temperature":21,"weekdays_p4_hour":19,"weekdays_p4_minute":0,"weekdays_p4_temperature":19},"running_state":"heat","sensor":"IN","system_mode":"heat"}'
debug 2023-02-28 13:56:06: Received Zigbee message from 'Door Sensor 2', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID null
info  2023-02-28 13:56:06: MQTT publish: topic 'zigbee2mqtt/Door Sensor 2', payload '{"battery":70,"contact":true,"device_temperature":4,"linkquality":255,"power_outage_count":21143,"temperature":23,"voltage":2955}'
debug 2023-02-28 13:56:06: Received Zigbee message from 'Termostat Szary', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,22],"type":"Buffer"},"datatype":2,"dp":16}],"seq":25859}' from endpoint 1 with groupID null
info  2023-02-28 13:56:06: MQTT publish: topic 'zigbee2mqtt/Termostat Szary', payload '{"child_lock":"UNLOCK","current_heating_setpoint":22,"deadzone_temperature":1,"heat":"ON","linkquality":255,"local_temperature":21,"local_temperature_calibration":-2,"max_temperature":35,"max_temperature_limit":26,"min_temperature_limit":null,"preset":"program","preset_mode":"program","program":{"saturday_p1_hour":6,"saturday_p1_minute":0,"saturday_p1_temperature":21,"saturday_p2_hour":12,"saturday_p2_minute":0,"saturday_p2_temperature":21,"saturday_p3_hour":14,"saturday_p3_minute":0,"saturday_p3_temperature":21,"saturday_p4_hour":19,"saturday_p4_minute":0,"saturday_p4_temperature":19,"sunday_p1_hour":8,"sunday_p1_minute":0,"sunday_p1_temperature":21,"sunday_p2_hour":12,"sunday_p2_minute":0,"sunday_p2_temperature":21,"sunday_p3_hour":14,"sunday_p3_minute":0,"sunday_p3_temperature":21,"sunday_p4_hour":19,"sunday_p4_minute":0,"sunday_p4_temperature":19,"weekdays_p1_hour":4,"weekdays_p1_minute":30,"weekdays_p1_temperature":21,"weekdays_p2_hour":12,"weekdays_p2_minute":0,"weekdays_p2_temperature":21,"weekdays_p3_hour":15,"weekdays_p3_minute":0,"weekdays_p3_temperature":21,"weekdays_p4_hour":19,"weekdays_p4_minute":0,"weekdays_p4_temperature":19},"running_state":"heat","sensor":"IN","system_mode":"heat"}'
debug 2023-02-28 13:56:06: Received Zigbee message from 'Termostat Szary', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,22],"type":"Buffer"},"datatype":2,"dp":16}],"seq":26115}' from endpoint 1 with groupID null
info  2023-02-28 13:56:06: MQTT publish: topic 'zigbee2mqtt/Termostat Szary', payload '{"child_lock":"UNLOCK","current_heating_setpoint":22,"deadzone_temperature":1,"heat":"ON","linkquality":255,"local_temperature":21,"local_temperature_calibration":-2,"max_temperature":35,"max_temperature_limit":26,"min_temperature_limit":null,"preset":"program","preset_mode":"program","program":{"saturday_p1_hour":6,"saturday_p1_minute":0,"saturday_p1_temperature":21,"saturday_p2_hour":12,"saturday_p2_minute":0,"saturday_p2_temperature":21,"saturday_p3_hour":14,"saturday_p3_minute":0,"saturday_p3_temperature":21,"saturday_p4_hour":19,"saturday_p4_minute":0,"saturday_p4_temperature":19,"sunday_p1_hour":8,"sunday_p1_minute":0,"sunday_p1_temperature":21,"sunday_p2_hour":12,"sunday_p2_minute":0,"sunday_p2_temperature":21,"sunday_p3_hour":14,"sunday_p3_minute":0,"sunday_p3_temperature":21,"sunday_p4_hour":19,"sunday_p4_minute":0,"sunday_p4_temperature":19,"weekdays_p1_hour":4,"weekdays_p1_minute":30,"weekdays_p1_temperature":21,"weekdays_p2_hour":12,"weekdays_p2_minute":0,"weekdays_p2_temperature":21,"weekdays_p3_hour":15,"weekdays_p3_minute":0,"weekdays_p3_temperature":21,"weekdays_p4_hour":19,"weekdays_p4_minute":0,"weekdays_p4_temperature":19},"running_state":"heat","sensor":"IN","system_mode":"heat"}'
debug 2023-02-28 13:56:06: Received Zigbee message from 'Termostat Szary', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,22],"type":"Buffer"},"datatype":2,"dp":16}],"seq":26371}' from endpoint 1 with groupID null
info  2023-02-28 13:56:06: MQTT publish: topic 'zigbee2mqtt/Termostat Szary', payload '{"child_lock":"UNLOCK","current_heating_setpoint":22,"deadzone_temperature":1,"heat":"ON","linkquality":255,"local_temperature":21,"local_temperature_calibration":-2,"max_temperature":35,"max_temperature_limit":26,"min_temperature_limit":null,"preset":"program","preset_mode":"program","program":{"saturday_p1_hour":6,"saturday_p1_minute":0,"saturday_p1_temperature":21,"saturday_p2_hour":12,"saturday_p2_minute":0,"saturday_p2_temperature":21,"saturday_p3_hour":14,"saturday_p3_minute":0,"saturday_p3_temperature":21,"saturday_p4_hour":19,"saturday_p4_minute":0,"saturday_p4_temperature":19,"sunday_p1_hour":8,"sunday_p1_minute":0,"sunday_p1_temperature":21,"sunday_p2_hour":12,"sunday_p2_minute":0,"sunday_p2_temperature":21,"sunday_p3_hour":14,"sunday_p3_minute":0,"sunday_p3_temperature":21,"sunday_p4_hour":19,"sunday_p4_minute":0,"sunday_p4_temperature":19,"weekdays_p1_hour":4,"weekdays_p1_minute":30,"weekdays_p1_temperature":21,"weekdays_p2_hour":12,"weekdays_p2_minute":0,"weekdays_p2_temperature":21,"weekdays_p3_hour":15,"weekdays_p3_minute":0,"weekdays_p3_temperature":21,"weekdays_p4_hour":19,"weekdays_p4_minute":0,"weekdays_p4_temperature":19},"running_state":"heat","sensor":"IN","system_mode":"heat"}'
debug 2023-02-28 13:56:06: Received Zigbee message from 'Termostat Szary', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,22],"type":"Buffer"},"datatype":2,"dp":16}],"seq":26627}' from endpoint 1 with groupID null
info  2023-02-28 13:56:06: MQTT publish: topic 'zigbee2mqtt/Termostat Szary', payload '{"child_lock":"UNLOCK","current_heating_setpoint":22,"deadzone_temperature":1,"heat":"ON","linkquality":255,"local_temperature":21,"local_temperature_calibration":-2,"max_temperature":35,"max_temperature_limit":26,"min_temperature_limit":null,"preset":"program","preset_mode":"program","program":{"saturday_p1_hour":6,"saturday_p1_minute":0,"saturday_p1_temperature":21,"saturday_p2_hour":12,"saturday_p2_minute":0,"saturday_p2_temperature":21,"saturday_p3_hour":14,"saturday_p3_minute":0,"saturday_p3_temperature":21,"saturday_p4_hour":19,"saturday_p4_minute":0,"saturday_p4_temperature":19,"sunday_p1_hour":8,"sunday_p1_minute":0,"sunday_p1_temperature":21,"sunday_p2_hour":12,"sunday_p2_minute":0,"sunday_p2_temperature":21,"sunday_p3_hour":14,"sunday_p3_minute":0,"sunday_p3_temperature":21,"sunday_p4_hour":19,"sunday_p4_minute":0,"sunday_p4_temperature":19,"weekdays_p1_hour":4,"weekdays_p1_minute":30,"weekdays_p1_temperature":21,"weekdays_p2_hour":12,"weekdays_p2_minute":0,"weekdays_p2_temperature":21,"weekdays_p3_hour":15,"weekdays_p3_minute":0,"weekdays_p3_temperature":21,"weekdays_p4_hour":19,"weekdays_p4_minute":0,"weekdays_p4_temperature":19},"running_state":"heat","sensor":"IN","system_mode":"heat"}'
debug 2023-02-28 13:56:06: Received Zigbee message from 'Termostat Szary', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,22],"type":"Buffer"},"datatype":2,"dp":16}],"seq":26883}' from endpoint 1 with groupID null
info  2023-02-28 13:56:06: MQTT publish: topic 'zigbee2mqtt/Termostat Szary', payload '{"child_lock":"UNLOCK","current_heating_setpoint":22,"deadzone_temperature":1,"heat":"ON","linkquality":255,"local_temperature":21,"local_temperature_calibration":-2,"max_temperature":35,"max_temperature_limit":26,"min_temperature_limit":null,"preset":"program","preset_mode":"program","program":{"saturday_p1_hour":6,"saturday_p1_minute":0,"saturday_p1_temperature":21,"saturday_p2_hour":12,"saturday_p2_minute":0,"saturday_p2_temperature":21,"saturday_p3_hour":14,"saturday_p3_minute":0,"saturday_p3_temperature":21,"saturday_p4_hour":19,"saturday_p4_minute":0,"saturday_p4_temperature":19,"sunday_p1_hour":8,"sunday_p1_minute":0,"sunday_p1_temperature":21,"sunday_p2_hour":12,"sunday_p2_minute":0,"sunday_p2_temperature":21,"sunday_p3_hour":14,"sunday_p3_minute":0,"sunday_p3_temperature":21,"sunday_p4_hour":19,"sunday_p4_minute":0,"sunday_p4_temperature":19,"weekdays_p1_hour":4,"weekdays_p1_minute":30,"weekdays_p1_temperature":21,"weekdays_p2_hour":12,"weekdays_p2_minute":0,"weekdays_p2_temperature":21,"weekdays_p3_hour":15,"weekdays_p3_minute":0,"weekdays_p3_temperature":21,"weekdays_p4_hour":19,"weekdays_p4_minute":0,"weekdays_p4_temperature":19},"running_state":"heat","sensor":"IN","system_mode":"heat"}'

```
Seems that after shutting down power at entire home and turning it back on the network is back in more or less normal. I've also disconnected for a while the termostat from power. After powering it back on it still sends a lot of messages, on average 50 messages per minute. Could the device be faulty? Or was it always behaving this way and just this morning the network got stuck with it?

**Comment by adorobis** [comment]
BTW, I've just noticed that the message flooding is a known but of this device so I've added the debounce option now. This indeed reduces the MQTT broker from the flood of messages but the network still gets them all. Not sure if there is any way of limiting it / limiting the impact on the network? For this issue the MQTT was coping with the load, it was the zigbee network that was not able to process them (or was it the zigbee2mqtt?).
